CIME/SystemTests/mvk.py

```diff
@@ -9,35 +9,129 @@
 import os
 import json
 import logging
-
 from distutils import dir_util
 
-import CIME.test_status
-import CIME.utils
+from CIME import test_status
+from CIME import utils
 from CIME.SystemTests.system_tests_common import SystemTestsCommon
 from CIME.case.case_setup import case_setup
 from CIME.XML.machines import Machines
-
+from CIME.config import ConfigBase
+from CIME.SystemTests import test_mods
 
 import evv4esm  # pylint: disable=import-error
 from evv4esm.__main__ import main as evv  # pylint: disable=import-error
 
-evv_lib_dir = os.path.abspath(os.path.dirname(evv4esm.__file__))
+version = evv4esm.__version_info__
+
+assert version[0] <= 0 and version[1] <= 5, "Please install evv4esm less than 0.5.x"
+
+EVV_LIB_DIR = os.path.abspath(os.path.dirname(evv4esm.__file__))
+
 logger = logging.getLogger(__name__)
-NINST = 30
+
+
+class MVKConfig(ConfigBase):
+    def __init__(self):
+        super().__init__()
+
+        if self.loaded:
+            return
+
+        self._set_attribute("component", "", "The main component.")
+        self._set_attribute(
+            "components", [], "Components that require namelist customization."
+        )
+        self._set_attribute("ninst", 30, "The number of instances.")
+        self._set_attribute(
+            "critical", 13, "The critical value for rejecting the null hypothese."
```

Comment:
EVV no longer needs this as of v0.5.0, (implemented false discovery rate correction), but that's my fault as it's not fully documented! It won't cause an error, but it isn't used by evv's `ks.py`